package.json

```diff
@@ -31,16 +31,16 @@
     "lint-fix": "yarn lint --fix",
     "release": "yarn clean && yarn build && standard-version && git push --follow-tags && npm publish",
     "start": "nuxt-ts start demo",
-    "test": "yarn lint && yarn build && nuxt build test/fixture && jest"
```

Comment:
Why remove the lint?

Reply:
Oops, my mistake. I was trying to test, but lint errors were preventing jest to run.

Reply:
Yeah that makes sense. Do you want to try seeing if downgrading eslint fixes yarn lint for now? See my other comment.